ffsync/ordered_id_test.go

```diff
@@ -33,16 +39,131 @@ func TestUint64OrderedId(t *testing.T) {
 }
 
 func TestMemoryIdGenerator(t *testing.T) {
```

Comment:
You could write a single test that takes an ID generater and t *testing.T then each of these specific tests just calls that. Otherwise you're copy and pasting a ton